dps-web-experience-sample/Views/Shared/_DocsNav.cshtml

```diff
@@ -193,15 +191,15 @@
 
             <option value="~/en-us/docs/cordova-5-security/" id="cordova-5-security-option">Cordova 5 security</option>
 
-            <option value="~/">Configure your app</option>
+            <option value="~/en-us/docs/configure-app/" id="configure-app">Configure your app</option>
```

Comment:
Thanks Raghav – I’ll fix it!

From: Raghav Katyal [mailto:notifications@github.com]
Sent: Monday, September 14, 2015 6:28 PM
To: rakatyal/vs-cordova-docs-sample <vs-cordova-docs-sample@noreply.github.com>
Cc: Norm Estabrook <normesta@microsoft.com>
Subject: Re: [vs-cordova-docs-sample] Updating TOC with link targets (#3)


In dps-web-experience-sample/Views/Shared/_DocsNav.cshtml<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2frakatyal%2fvs-cordova-docs-sample%2fpull%2f3%23discussion_r39466385&data=01%7c01%7cnormesta%40exchange.microsoft.com%7cceb5b72d5fa24e7a358108d2bd6cf5ec%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=qViDoy0Pt%2fWeCaWEn6DAeD792PgNc2kq1%2birL6srKIo%3d>:

> @@ -193,15 +191,15 @@

>

>              <option value="~/en-us/docs/cordova-5-security/" id="cordova-5-security-option">Cordova 5 security</option>

>

> -            <option value="~/">Configure your app</option>

> +            <option value="~/en-us/docs/configure-app/" id="configure-app">Configure your app</option>

Hi Norm! IDs have to be unique on each element of the page. These ones(small nav element) have "option" appended. So for "configure-app" document, id will be "configure-app-option" and so on. For the
•  elements above they are the same as the document id.

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2frakatyal%2fvs-cordova-docs-sample%2fpull%2f3%2ffiles%23r39466385&data=01%7c01%7cnormesta%40exchange.microsoft.com%7cceb5b72d5fa24e7a358108d2bd6cf5ec%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=7WQmmwTI8Dlbiqtpm1UqJcyy4nRztzik2jO7Pi%2bf13o%3d>.